docs/01-install.md

```diff
@@ -189,52 +189,76 @@ authUser log,execute,net trapuser
 
 Edit the launch options of snmptrapd
 ------------------------
+In order for trapdirector to receive traps in the correct format, snmptrapd startup options must include `-n -Oen`. Here is how to enable these options on various systems:
 
-* For RH7/CenOS7 and other systems using systemd : 
-
-In `/usr/lib/systemd/system/snmptrapd.service` or `/lib/systemd/system/snmptrapd.service` 
-
-If you have a line like `EnvironmentFile=-/etc/sysconfig/snmptrapd` change that file instead (as you would in RH6 example below).
+On RHEL6 based distros, `/etc/sysconfig/snmptrapd` should contain:
+```
+OPTIONS="-Lsd -n -t -Oen -p /var/run/snmptrapd.pid"
+```
 
-If not, change `Environment=OPTIONS="-Lsd"` to `Environment=OPTIONS="-Lsd -n -t -Oen"`
+On newer RHEL based distros, `/etc/sysconfig/snmptrapd` should contain:
+```
+OPTIONS="-Lsd -n -t -Oen"
+```
 
-If you have a weird 204 error on startup (happened on CentOS7 system), change `ExecStart` line instead to 
+On Debian based distros with systemd:
+1. Copy the systemd snmptrapd service:
+```
+cp -v /lib/systemd/system/snmptrapd.service /etc/systemd/system/
+```
+2. Edit `/etc/systemd/system/snmptrapd.service` to contain:
+```
+ExecStart=/usr/sbin/snmptrapd -Lsd -n -t -Oed -f
+```
 
-`ExecStart=/usr/sbin/snmptrapd -n -t -Oen $OPTIONS -f`
+On Debian based distros with System V-style init, `/etc/default/snmptrapd` should contain:
+```
+TRAPDRUN=yes
 
+TRAPDOPTS='-Lsd -n -t -Oen -p /run/snmptrapd.pid'
+```
 
-* For RH6/CenOS6 and other /etc/init.d system services 
+On other systems, check:
+* `/etc/sysconfig/snmptrapd`
+* `/etc/default/snmptrapd`
+* `/etc/init.d/snmptrapd`
+* `Environment` line in `/usr/lib/systemd/system/snmptrapd.service` or `/lib/systemd/system/snmptrapd.service`
+* `ExecStart` line in `/usr/lib/systemd/system/snmptrapd.service` or `/lib/systemd/system/snmptrapd.service`
 
-In `/etc/sysconfig/snmptrapd` file change `# OPTIONS="-Lsd -p /var/run/snmptrapd.pid"` line to `OPTIONS="-Lsd -n -t -Oen -p /var/run/snmptrapd.pid"`
+If you can't find any of these files, it's likely you need to install the `net-snmp` or `snmptrapd` package.
 
 Enable & start snmptrap service : 
 ------------------------
 
 * On systemd:
-
 ```
 systemctl daemon-reload
 
 systemctl enable snmptrapd
 
-systemctl start snmptrapd
+systemctl restart snmptrapd
```

Comment:
Yes, I thought it could avoid some troubleshooting / confusion, as "restart" works even if the service is not already started, under both `systemctl` and `service`.